bin/package/installation/systemd.service

```diff
@@ -14,7 +14,7 @@ RuntimeDirectoryMode=0750
 LogsDirectory=mysterium-node
 
 EnvironmentFile=-/etc/default/mysterium-node
-ExecStart=/usr/bin/myst $CONF_DIR $DATA_DIR $RUN_DIR $DAEMON_OPTS service $SERVICE_OPTS --agreed-terms-and-conditions
+ExecStart=/usr/bin/myst $CONF_DIR $DATA_DIR $RUN_DIR $BUILTIN_UI $DAEMON_OPTS service $SERVICE_OPTS --agreed-terms-and-conditions
```

Comment:
Hey, lets not add more vars, everything is doable with `$DAEMON_OPTS` and `$SERVICE_OPTS`

Reply:
Removed it at all, UI is not working now, we will enable it a bit later.